resources/leiningen/new/kraken/core.clj

```diff
@@ -1,19 +1,17 @@
 (ns {{name}}.core
-    (:require [kehaar :as k]
-              [langohr.consumers :as lc]
+    (:require [{{name}}.channels :as channels]
               [{{name}}.queue :as queue]
               [turbovote.resource-config :refer [config]]
-              [clojure.tools.logging :as log]))
-
-(defn handler
-  "A handler that does nothing and responds ok."
-  [message]
-  (log/info "Received:" message)
-  {:status :ok})
+              [datomic-toolbox :as db]
+              [clojure.tools.logging :as log]
+              [immutant.util :as immutant]))
 
 (defn -main [& args]
-  (queue/initialize)
-  (lc/subscribe @queue/channel
-                (config :rabbit-mq :queue)
-                (k/simple-responder handler)
-                {:auto-ack true}))
+  (when (config :datomic :intitialize)
+    (db/initialize))
+  (when (config :datomic :run-migrations)
```

Comment:
Butting in here...run-migrations is part of db/initialize, so instead of (when (config :datomic :initialize)...) I think it should be an if, and then move the when :run-migrations into the else form.